Libplanet/Net/Swarm.cs

```diff
@@ -1650,107 +1641,129 @@ CancellationToken cancellationToken
         )
         {
             BlockChain<T> workspace = blockChain;
+            List<Guid> scope = new List<Guid>();
```

Comment:
Trivial, just for consistency.

```suggestion
            var scope = new List<Guid>();
```